packages/rest/src/__tests__/unit/rest.server/rest.server.unit.ts

```diff
@@ -96,7 +96,7 @@ describe('RestServer', () => {
 
     it('honors gracePeriodForClose', async () => {
       const app = new Application({
-        rest: {gracePeriodForClose: 1000},
+        rest: {port: 0, gracePeriodForClose: 1000},
```

Comment:
is `port:0` to make it find a suitable port number ?

Reply:
Yes. Otherwise it uses `3000`, which may conflict with other apps.